lib/school_house_web/router.ex

```diff
@@ -25,14 +25,17 @@ defmodule SchoolHouseWeb.Router do
     get "/blog/:slug", PostController, :show
 
     get "/privacy", PageController, :privacy
-    get "/podcasts", PageController, :podcasts
-    get "/conferences", PageController, :conferences
-
-    get "/:locale", PageController, :index
-    get "/:locale/report", ReportController, :index
-    get "/:locale/why", PageController, :why
-    get "/:locale/:section", LessonController, :index
-    get "/:locale/:section/:name", LessonController, :lesson
+
+    scope "/:locale" do
+      get "/", PageController, :index
+      get "/report", ReportController, :index
+      get "/why", PageController, :why
+      get "/podcasts", PageController, :podcasts
+      get "/conferences", PageController, :conferences
```

Comment:
```suggestion
      get "/", PageController, :index
      get "/why", PageController, :why
      get "/podcasts", PageController, :podcasts
      get "/conferences", PageController, :conferences
      
       get "/report", ReportController, :index
```

The lesson controller methods are separated out, perhaps it would be nice to separate out the reports controller method as well.

Reply:
Sounds good! Thanks @kinson  👍